masterserver/Dockerfile

```diff
@@ -9,8 +9,10 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 # install runtime deps
 RUN apt-get update && apt-get install -y \
-    software-properties-common curl autoconf git apt-utils libgit2-dev libio-captureoutput-perl python python-pip python3-distutils protobuf-compiler icu-devtools libicu-dev && \
+    software-properties-common curl autoconf git apt-utils libgit2-dev libio-captureoutput-perl unzip python python-pip python3-distutils icu-devtools libicu-dev && \
     rm -rf /var/lib/apt/lists/*
+RUN curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.11.4/protoc-3.11.4-linux-x86_64.zip
```

Comment:
Why do we need this? Please explain in your commit message why not to keep the previous installation.